lib/main.dart

```diff
@@ -28,6 +31,16 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   TextStyle style = TextStyle(fontFamily: 'Montserrat', fontSize: 20.0);
 
+  final _emailTextController = TextEditingController();
+  final _passwordTextController = TextEditingController();
+
+  @override
+  void dispose(){
```

Comment:
Is this implicitly called at some point? I don't see any calls to it

Reply:
Yes, dispose is implicitly called when the widget is disposed (i.e. going to new screen) but TextEditingControllers are not automatically disposed when their parent widget is disposed. So it is necessary to override the widget's dispose to also dispose of the controllers.

Reply:
Nice, I didn't know that.